.travis.yml

```diff
@@ -1,9 +1,5 @@
 # http://travis-ci.org/#!/gjhiggins/rdflib-sqlalchemy
 language: python
-branches:
```

Comment:
don't think we should be limiting when to run CI, esp. if we adopt something like the git-flow model that uses feature branches more extensively. It's possible to exclude certain branches to form a 'blacklist' which might be useful in some cases